metadata_backend/api/operators.py

```diff
@@ -336,6 +421,21 @@ async def _format_data_to_create_and_add_to_db(self, schema_type: str,
                                                {"accessionId": accession_id,
                                                 "content": data}))
 
+    async def _format_data_to_replace_and_add_to_db(self, schema_type: str,
+                                                    accession_id: str,
+                                                    data: str) -> str:
+        """Format XML metadata object and add it to db.
+
+        XML is validated, then parsed to json and json is added to database.
+        After successful json insertion, xml itself is backed up to database.
+
+        :param schema_type: Schema type of the object to replace.
+        :param accession_id: Identifier of object to replace.
+        :param data: Original xml content
+        :returns: Accession Id for object inserted to database
+        """
+        raise web.HTTPNotImplemented
```

Comment:
Why do we add this if we're not implementing it?

Reply:
`_format_data_to_replace_and_add_to_db()` was set as an abstract method so I'm guessing this is here so that the syntax is not broken. If the idea is to implement this later, then I'm fine with it being like this for now.

Reply:
@csc-jm is right  also we will use that when implementing XML file type replace in #21

Reply:
ok, no problem, I was just wondering why to add it before implementing.